server/routes/routes.js

```diff
@@ -430,19 +423,17 @@ router.post('/', async (req, res) => {
       const endPolygon = await fetchDestinationRadius(endCoordinates);
 
       if (startPolygon && endPolygon) {
+        console.log(startPolygon)
```

Comment:
Are these logs just for debugging? I think it's ok if you want to leave some detailed console logs for as a way to make sure the important tasks are still working as expected, e.g. it could be a sanity check/testing method for your project.

Reply:
Yes, they were a temporary fix for the _undefined_ errors. They have been taken out in future PRs